docs/operating-scylla/nodetool-commands/backup.rst

```diff
@@ -10,7 +10,8 @@ Syntax
 .. code-block:: console
 
    nodetool [(-h <host> | --host <host>)] [(-p <port> | --port <port>)]
-               --keyspace <keyspace> [--snapshot <snapshot>]
+               --keyspace <keyspace> --table <table>
+               [--snapshot <snapshot>]
```

Comment:
it's a pre-existing issue, but shouldn't the first line say `nodetool backup` rather than just `nodetool`?

Reply:
Also, the `--prefix` option is missing here.